<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Explore</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QTreeView" name="treeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Setting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="SettingsWidget" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>MySetting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="MySettingWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Unknow</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>12</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>43</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="accelerated">
         <bool>false</bool>
        </property>
        <property name="minimumDate">
         <date>
          <year>1752</year>
          <month>10</month>
          <day>14</day>
         </date>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>44</x>
          <y>72</y>
          <width>104</width>
          <height>26</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>New Item</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>104</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>dddd/dd/dd</string>
        </property>
        <property name="text">
         <string>212/1/2</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>150</y>
          <width>26</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>150</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>190</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>dddddddddd</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>113</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>110</y>
          <width>48</width>
          <height>24</height>
         </rect>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>220</y>
          <width>87</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="JSWidget" name="widget_3" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>settingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MySettingWidget</class>
   <extends>QWidget</extends>
   <header>mysettingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JSWidget</class>
   <extends>QWidget</extends>
   <header>jswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
